src/components/dialogs/contingency-list-helper.js

```diff
@@ -9,7 +9,7 @@ export const getIdentifierContingencyListFromResponse = (response) => {
     const result = response?.identifierContingencyList?.identifiers?.map(
         (identifiers, index) => {
             return {
-                contingencyName: 'contingencyName' + index, // Temporary : at the moment, we do not save the name in the backend.
+                contingencyName: identifiers.contingencyId,
                 equipmentIDs: identifiers.identifierList.map(
```

Comment:
- That's not clear why we put the ID in the Name field.
- Maybe rename equipmentIDs to identifierList ?